pyracing/client.py

```diff
@@ -744,7 +744,12 @@ async def total_registered_all(self):
         url = ct.URL_TOTALREGISTERED
         response = await self._build_request(url, payload)
 
-        return [upcoming_events.TotalRegistered(x) for x in response.json()]
+        # Add missing double quotes for JSON accepted format.
+        t = response.text.replace('registered', '"registered"')
```

Comment:
lets make this into a helper method, something that takes in a string and a substring and surrounds the substring with quotes on either end. Will make this more reusable since we need to do it in a few places. Maybe we could even give it a string and a list of substrings and let it fix up the whole json string, that would make it way cleaner if we had one where we need to replace a bunch of these in the same payload.